npps4/webui/endpoints/list_users.py

```diff
@@ -0,0 +1,25 @@
+import fastapi
+import asyncio
+
+from .. import template
+from ...app import webui
+
+import sqlalchemy
+
+import npps4.db.main
+import npps4.idol
```

Comment:
Do relative import for NPPS4 package if it's inside `npps4` directory.
```suggestion
from ... import idol
from ...db import main
```